### ISSUE TITLE
`stack ghci` fails to load modules with spaces in path.

### ISSUE THREAD

**Issue by VitorCBSB** [commercialhaskell/stack#1401]
Windows 10.
```
C:\Users\Vitor Coimbra\img2ascii>stack --version
Version 0.1.8.0, Git revision 52bb19764e20a4433f3f8c65e4093d2a51568c00 (2602 commits) x86_64
```

Hey there, while playing around with stack, I noticed the command `stack ghci` does not load files with spaces in their paths.

For example, the module `C:\Users\Vitor Coimbra\img2ascii\src\Main.hs` fails to load with the following messages:

```
C:\Users\Vitor Coimbra\img2ascii>stack ghci
Setting codepage to UTF-8 (65001) to ensure correct output from GHC
Using main module: Package `img2ascii' component exe:img2ascii with main-is file: C:\Users\Vitor Coimbra\img2ascii\src\Main.hs
Configuring GHCi with the following packages: img2ascii
GHCi, version 7.10.2: http://www.haskell.org/ghc/  :? for help
ghc.exe: warning: _tzset from msvcrt is linked instead of __imp__tzset
target `C:\Users\Vitor' is not a module name or a source file
syntax:  :module [+/-] [*]M1 ... [*]Mn
Prelude>
```

and so, none of the functions in `Main` are available in the interpreter. Once again, I don't know if I missed some obvious step, but I can reproduce it by doing:

(Starts in the `C:\Users\Vitor Coimbra` directory)
```
stack new Test simple
cd Test
stack build
stack ghci
```

**Comment by mgsloan** [comment]
Good catch, thanks!  Fixed in https://github.com/commercialhaskell/stack/commit/0c5b396a50552c5a0b4a0bca9dd41dc6adb6ac07

**Comment by VitorCBSB** [comment]
Hey, thanks for fixing it. Works great!

The only thing I'm worried about is the syntax message is still there. I don't know if it's something I need to be worried about or not, but here it is.

```
C:\Users\Vitor Coimbra\img2ascii>stack --version
Version 0.1.9.0, Git revision cec298802573296f27eb0b0b09ed153e16766519 (2658 commits) x86_64

C:\Users\Vitor Coimbra\img2ascii>stack ghci
Using main module: Package `img2ascii' component exe:img2ascii with main-is file: C:\Users\Vitor Coimbra\img2ascii\src\Main.hs
Configuring GHCi with the following packages: img2ascii
GHCi, version 7.10.2: http://www.haskell.org/ghc/  :? for help
ghc.exe: warning: _tzset from msvcrt is linked instead of __imp__tzset
[1 of 1] Compiling Main             ( C:\Users\Vitor Coimbra\img2ascii\src\Main.hs, interpreted )
Ok, modules loaded: Main.
syntax:  :module [+/-] [*]M1 ... [*]Mn
*Main>
```

**Comment by mgsloan** [comment]
Good point, thanks for following up with that concern.  Fixed in 45a2f5d2d5faa4a3f79ef01f02e55143d95914dd
